### DISCUSSION TITLE
【Unknown】The rope disappears immediately after use, even the superadmin

### DISCUSSION THREAD

**Discussion by lovelatiao** [Pryaxis/TShock#1976] [Q&A]
When I place the rope, it's consumed, but it disappears as soon as it's placed.I don't know what happened to it.
  I don't know if the problem is bug or permission.If it's a permission issue.I want to know which authority.
  Thanks for your help！！！

**Comment by lovelatiao** [comment]
The rope made by spiders can't be used either,but chains can be used

**Comment by hakusaro** [comment]
Just to check, are you on the latest pre-release? (10)? Do you have any debug logs? If you turn on `DebugLogs` (set to `true`) in `config.json` it would help us out a lot!

**Reply by lovelatiao** [reply]
I did what you said.
`Debug: Bouncer / OnTileEdit rejected from (inconceivable rope coil) helloworld PlaceTile 213`

**Comment by hakusaro** [comment]
This is being tracked in #1980 now. We'll fix it!
